FEWD/Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,87 @@
+var humanScore = 0;
+var computerScore = 0;
+
+function compChoice() {
+  var computerChoice = Math.random();
```

Comment:
Great job using `Math.random` and the conditionals below to randomise the computer's choice!